db/seeds.rb

```diff
@@ -1 +1,61 @@
 # frozen_string_literal: true
+
+puts "Creating test snap application..."
+
```

Comment:
_not blocking_ - Does this make sense to make a factory?

Reply:
You *could* do that, sure. I don't have any strong feelings towards making this use factory girl. However, that would require pulling in FG to the development group, and maintaining it over in that context is a jump away.

In truth, I have no argument for or against it. What about you, @zspencer?

Reply:
Yeah I'm fine leaving it as a normal model. One of the things I was thinking about for testing was to add to the admin dashboard  SNAPApplication generator which could use factory girl traits and what not to build an app; but I don't think that's necessary for now.